### PR TITLE
fix windows build all.json broken

### PR DESCRIPTION
**Pull request by liudonghua123**
## References

try to fix https://github.com/jtpio/jupyterlite/issues/76.

### CODE REVIEW

File: app/webpack.config.js [hunk 1]
```diff
@@ -35,7 +35,8 @@ fs.copySync(
 );
 
 // ensure all schemas are statically compiled
-const schemaDir = path.resolve(buildDir, './schemas');
+// fix windows path issue, use / as path separator instead of \
+const schemaDir = path.resolve(buildDir, './schemas').replace(/\\/g, '/');
```
Comment (bollwyvl) [inline app/webpack.config.js:39]: perhaps we use [`path.posix`](https://nodejs.org/api/path.html#path_path_posix) everywhere? There will be more of these as we add things like static files, federated extensions, etc.

Comment (liudonghua123) [reply app/webpack.config.js:39]: I tried [path.posix](https://nodejs.org/api/path.html#path_path_posix), the result of `path.resolve` use posix path separator indeed, however it reduced the driver letter. So [path.posix](https://nodejs.org/api/path.html#path_path_posix) will not fix this problem here.

```js
> buildDir = './build'
'./build'
> require('path').resolve(buildDir, './schemas')
'D:\\code\\python\\jupyterlite\\build\\schemas'
> require('path/posix').resolve(buildDir, './schemas')
'/code/python/jupyterlite/build/schemas'
>
```

**Comment by bollwyvl** [comment]
Welp, any time we can lean on stdlib stuff rather than treating paths as
strings is a win... If we do have to, we should hoist it to a function e.g.
normalizePath...

**Comment by liudonghua123** [comment]
> Welp, any time we can lean on stdlib stuff rather than treating paths as strings is a win... If we do have to, we should hoist it to a function e.g. normalizePath...

Wrap the path operation using a function like `normalizePath` is a good choice, it makes the code more readable. I will try to patch the pr.

**Comment by bollwyvl** [comment]
This is now under test in [CI](https://github.com/jupyterlite/jupyterlite/actions/workflows/win-dev.yml), so presumably this is (and will stay) fixed.

**Pull request closed without merging**